File: feature-widget/src/main/java/home/oleg/feature_widget/PlaceRemoteService.java
package home.oleg.feature_widget;

import android.content.Intent;
import android.widget.RemoteViewsService;

public class PlaceRemoteService extends RemoteViewsService {

    @Override
    public RemoteViewsFactory onGetViewFactory(Intent intent) {
        return new ListRemoteViewsFactory(getApplicationContext(), intent);
    }
}

File: feature-widget/src/main/java/home/oleg/feature_widget/PlaceRemoteService.kt
package home.oleg.feature_widget

import android.content.Intent
import android.widget.RemoteViewsService

class PlaceRemoteService : RemoteViewsService() {

    override fun onGetViewFactory(intent: Intent): RemoteViewsService.RemoteViewsFactory {
        return ListRemoteViewsFactory(applicationContext, intent)
    }
}
